src/main/java/com/example/simplefileshare/simplefileshare/services/DeleteInterface.java

```diff
@@ -0,0 +1,7 @@
+package com.example.simplefileshare.simplefileshare.services;
+
+import java.net.URL;
+
+public interface DeleteInterface {    
```

Comment:
Can we merge UploadInterface and DeleteInterface into one interface like RemoteFileInterface? In the future, if we want to introduce more functionality we will have to create more interfaces in the current implementation.

Reply:
Yeah, I also thought the same thing... Will merge it.